src/js/options.js

```diff
@@ -69,7 +71,7 @@ function updateTabsVisibility(data) {
 }
 
 function mainConfigurationUpdated(lazy) {
-  if (!lazy) {
+  if (!lazy || !_.get(lazy, "detail")) {
```

Comment:
`lazy` should be a boolean flag, I don't understand this change (since in all the code we only do either `mainConfigurationUpdated()` or `mainConfigurationUpdated(true)`).

Reply:
@jfmdev when settings.svelte fires updateMainConfiguration event, the parameter is wrapped with event object's "detail" field. So, even I send false as parameter, because lazy will be something like {..., ... , detail: false, ...}, it will not enter to if block.

Reply:
Please add a comment explaining that.